osbuild/__init__.py

```diff
@@ -75,16 +75,55 @@ def __init__(self, path="/run/osbuild"):
         self.root = tempfile.mkdtemp(prefix="osbuild-buildroot-", dir=path)
         self.api = tempfile.mkdtemp(prefix="osbuild-api-", dir=path)
         self.mounted = False
+        self.usr_mounted = False
+        self.lib_mounted = False
+        self.lib64_mounted = False
+        self.bin_mounted = False
+        self.sbin_mounted = False
```

Comment:
These are all bind-mounted in the same way, right? Can we use a dictionary (path → mounted) here?

Reply:
Yup.

Reply:
Did it slightly differently in the end, but the same idea.